app/3.0/toolbox/prpl.md

```diff
@@ -72,32 +68,30 @@ When you generate an App Toolbox project using Polymer CLI, the new project cont
 
 The shell is responsible for routing and usually includes the main navigation UI for the app.
 
-The app should call `importHref` to lazy-load fragments as they're required. For example, when the
+The app should use [dynamic imports](https://developers.google.com/web/updates/2017/11/dynamic-import) to lazy-load fragments as they're required. For example, when the
 user changes to a new route, it imports the fragment(s) associated with that route. This may
 initiate a new request to the server, or simply load the resource from the cache.
 
-importHref example (class-style element) {.caption}
+Dynamic import example {.caption}
 
 ```js
-// get a URL relative to this element
-let resolvedUrl = this.resolveUrl('list-view.html');
-
-// import the file
-Polymer.importHref(
-    resolvedUrl,
-    null,  /* callback for successful load -- usually not needed */
-    this._importFailedCallback.bind(this), /* for example, display 404 page */
-    true); /* make import async */
-```
-
-importHref example (hybrid element) {.caption}
-
-```js
-var resolvedPageUrl = this.resolveUrl('my-' + page + '.html');
-this.importHref(resolvedPageUrl,
-    null,
-    this._importFailedCallback,
-    true);
+_pageChanged(page) {
+  // Load page import on demand. Show 404 page if fails
```

Comment:
This comment is out of sync with the code here: https://github.com/Polymer/polymer-starter-kit/blob/3.0-preview/src/my-app.js#L153

That branch, at least, is checking the page in `_routePageChanged`, and going to the 404 if the page isn't in the list of known pages. This code doesn't do anything if the import fails.